dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -1,137 +1,518 @@
+                          :showIcon="true"
+                          dateFormat="D, M dd, yy"
+                        />
+                      </div>
+
+                      <FormKit
+                        type="text"
+                        validation="required"
+                        validation-label="Fiscal year"
+                        name="fiscalYearEnd"
+                        v-model="convertedFiscalYearEnd"
+                        :outer-class="{ 'hidden-input': true }"
+                      />
+                    </div>
+
+                    <!-- Scope of entities -->
+                    <div class="form-field">
+                      <YesNoComponent
+                        :displayName="euTaxonomyKpiNameMappings.scopeOfEntities"
+                        :info="euTaxonomyKpiInfoMappings.scopeOfEntities"
+                        :name="'scopeOfEntities'"
+                      />
+                    </div>
+
+                    <!-- EU Taxonomy activity level reporting -->
+                    <div class="form-field">
+                      <YesNoComponent
+                        :displayName="euTaxonomyKpiNameMappings.activityLevelReporting"
+                        :info="euTaxonomyKpiInfoMappings.activityLevelReporting"
+                        :name="'activityLevelReporting'"
+                      />
+                    </div>
+
+                    <!-- Number of employees -->
+                    <div class="form-field">
+                      <UploadFormHeader
+                        :name="euTaxonomyKpiNameMappings.numberOfEmployees"
+                        :explanation="euTaxonomyKpiInfoMappings.numberOfEmployees"
+                      />
+                      <div class="lg:col-4 md:col-4 col-6 p-0">
+                        <FormKit
+                          type="number"
+                          name="numberOfEmployees"
+                          validation-label="Number of employees"
+                          placeholder="Value"
+                          validation="required|number"
+                          step="1"
+                          min="0"
+                        />
+                      </div>
+                    </div>
+
+                    <!-- EU Taxonomy activity level reporting -->
+                    <div class="form-field">
+                      <YesNoComponent
+                        :displayName="euTaxonomyKpiNameMappings.reportingObligation"
+                        :info="euTaxonomyKpiInfoMappings.reportingObligation"
+                        :name="'reportingObligation'"
+                      />
+                    </div>
+                  </div>
+                </div>
+
+                <div class="uploadFormSection">
+                  <div class="col-3 p-3 topicLabel">
+                    <h4 id="assurance" class="anchor title">Assurance</h4>
+                  </div>
+
+                  <!-- Level of assurance -->
```

Comment:
In the figma there is a field for NFRD, here i did not saw it. Is this intended?

Reply:
It is not included in our data model "CompanyAssociatedDataEuTaxonomyDataForFinancials" and this should be more important

Reply:
Ok, as discussed in the daily this will be reintroduced in the eu taxonomy rework ticket